source/change-streams/tests/unified/change-streams-errors.yml

```
@@ -0,0 +1,126 @@
+description: "change-streams-errors"
+schemaVersion: "1.7"
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+      ignoreCommandMonitoringEvents: [ killCursors ]
+      useMultipleMongoses: false
+  - client:
+      id: &globalClient globalClient
+      useMultipleMongoses: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+  - database:
+      id: &globalDatabase0 globalDatabase0
+      client: *globalClient
+      databaseName: *database0
+  - collection:
+      id: &globalCollection0 globalCollection0
+      database: *globalDatabase0
+      collectionName: *collection0
+
+tests:
+  - description: "The watch helper must not throw a custom exception when executed against a single server topology, but instead depend on a server error"
+    runOnRequirements:
+      - minServerVersion: "3.6.0"
+        topologies:
+          - single
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+        expectError:
+          errorCode: 40573
+  - description: Change Stream should error when an invalid aggregation stage is passed in
```

```suggestion

  - description: Change Stream should error when an invalid aggregation stage is passed in
```

IMO, newlines before each test (after the first) help with readability when editing YAML files. Feel free to disregard this suggestion, though.

---

Agreed - done.